tests/test_feedexport.py

```diff
@@ -2850,20 +2850,8 @@ def test_init(self):
             "FEED_URI": "file:///tmp/foobar",
             "FEED_STORAGES": {"file": StdoutFeedStorageWithoutFeedOptions},
         }
-        with pytest.warns(
-            ScrapyDeprecationWarning,
-            match="The `FEED_URI` and `FEED_FORMAT` settings have been deprecated",
-        ):
-            crawler = get_crawler(settings_dict=settings_dict)
-            feed_exporter = FeedExporter.from_crawler(crawler)
-
-        spider = scrapy.Spider("default")
-        with pytest.warns(
-            ScrapyDeprecationWarning,
-            match="StdoutFeedStorageWithoutFeedOptions does not support "
-            "the 'feed_options' keyword argument.",
-        ):
-            feed_exporter.open_spider(spider)
+        with pytest.raises(TypeError):
```

Comment:
Similarly, I think all these `*PreFeedOptionsTest` tests can be removed.

Reply:
Done.